node_test/rpc_test.go

```diff
@@ -143,6 +143,24 @@ func executeNRpcTest(t *testing.T, connectionType transport.TransportType, n int
 	}
 	t.Log("Ledger channels created")
 
+	// try to create duplicate ledger channel to ensure node correctly
+	// handles error without panicking
+	{
+		outcome := simpleOutcome(actors[0].Address(), actors[1].Address(), 100, 100)
+		duplicateLedgerChannel := clients[0].CreateLedgerChannel(actors[1].Address(), 100, outcome)
```

Comment:
This name is a bit confusing
```suggestion
		duplicateLedgerChannelObjective := clients[0].CreateLedgerChannel(actors[1].Address(), 100, outcome)
```